src/test/Components/CreateImageWizard/CreateImageWizard.test.js

```diff
@@ -48,9 +45,9 @@ let router = undefined;
 // to navigate to the images table (via useNavigate hook), which will in turn
 // result in a call to getComposes. If it is not mocked, tests fail due to MSW
 // being unable to resolve that endpoint.
-jest
-  .spyOn(api, 'getComposes')
-  .mockImplementation(() => Promise.resolve(mockComposesEmpty));
+// jest
```

Comment:
Why removing the lines in CreateImageWizard.content.test.js and ImagesTable.test.js, but only commenting them here?

Reply:
Oversight on my part I'm sure :sweat_smile: nice catch!